scram-tools.file/tools/pytorch-cluster/pytorch-cluster.xml

```diff
@@ -0,0 +1,10 @@
+<tool name="pytorch-cluster" version="@TOOL_VERSION@">
+  <client>
+    <environment name="TORCH_CLUSTER_BASE" default="@TOOL_ROOT@"/>
```

Comment:
@valsdav , can you please update all the newly added xml files  to have `PYTORCH_` instead of `TORCH_` e.g. replace `TORCH_CLUSTER_BASE` with `PYTORCH_CLUSTER_BASE`  every where. The name of this variable should match the name of tool.

Reply:
fixed in https://github.com/cms-sw/cmsdist/pull/9094/commits/492ec080954424dc5dae3fba178c76f8c73a2777